packages/buefy/src/components/autocomplete/Autocomplete.vue

```diff
@@ -253,28 +256,35 @@ export default defineComponent({
         },
         isEmpty() {
             if (!this.computedData) return true
-            return !this.computedData.some((element) => element.items && element.items.length)
+            return !this.computedData.some(
+                (element) => element.items && element.items.length
+            )
         },
         /*
          * White-listed items to not close when clicked.
          * Add input, dropdown and all children.
          */
         whiteList() {
+            // access computedData to make this property reactive when data changes
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            const _ = this.computedData
             const whiteList = []
-            whiteList.push((this.$refs.input as BInputComponent).$el.querySelector('input'))
+            whiteList.push(
+                (this.$refs.input as BInputComponent).$el.querySelector('input')
+            )
             whiteList.push(this.$refs.dropdown)
-            // Add all children from dropdown
             if (this.$refs.dropdown != null) {
-                const children = (this.$refs.dropdown as Element).querySelectorAll('*')
+                const children = (
```

Comment:
Actually, those formatting changes disturb reviewing because real changes are interleaved in stylistic ones. Perhaps it would be wise to minimize format changes and stick to the current formatting, and just commit the changes in "logic" next time. That would make it easier / faster for the reviewer ...just nitpicking, no biggy.